stock_picking_report_by_partner/views/partner_view.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+  <data>
+
+    <record id="view_partner_picking_report_form" model="ir.ui.view">
+        <field name="name">res.partner.stock.picking.report.form</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="product.view_partner_property_form"/>
+        <field name="arch" type="xml">
+            <group name="pricelists" position="after">
+                <group>
+                    <field name="picking_report_id"/>
```

Comment:
I think it can be usefull to have the context to set automatically the model stock.picking.